### ISSUE TITLE
Licence / domaine public

### ISSUE THREAD

**Issue by hugoroy** [steeve/france.code-civil#5]
Je ne comprends pas l'intérêt de mettre une licence creative commons sur quelque chose qui est du domaine public.

**Comment by lisael** [comment]
IANAL, mais le concept de domaine public n'existe pas dans le code civil ( un petit grep sur le projet pourrait le {con,in}firmer ). Ceci dit le problème de la licence reste. Je ne pense pas que tu puisse re-licencier ce texte.

**Comment by lisael** [comment]
http://rip.journal-officiel.gouv.fr/http://rip.journal-officiel.gouv.fr/ hum...

« signale que la réutilisation des données mises à disposition par la DILA est soumise au paiement d’une redevance ou à des conditions particulières. »

**Comment by hugoroy** [comment]
"IAAL" et les textes officiels comme les lois, les décisions de justice, etc ne sont pas soumis au droit d'auteur (donc ils sont "du domaine public"). C'est un principe qu'on retrouve dans la convention de Berne et qui est de jurisprudence en France. Je n'ai pas l'arrêt sous la main, mais c'est une évidence : sinon, comment un avocat pourrait-il travailler s'il n'avait pas le droit de reproduire les textes de lois et les décisions de justice dans ses écritures et ses publications ?

La réutilisation des données est un autre problème, ce n'est pas du droit d'auteur.

(voir, peut être http://rip.journal-officiel.gouv.fr/index.php/pages/LO )

**Comment by lisael** [comment]
Ok, c'est presque parfait, alors (en fonction de la source de donnée du projet). Ce qui peut être licenciable en CC c'est le travail de compilation des diffs, qui doit être énorme

**Comment by Daylon** [comment]
Dans le cas de la diffusion du droit Internet, il existe [l'article 4 du décret n°2002-1064](http://www.legifrance.gouv.fr/affichTexteArticle.do;jsessionid=BB688D2D2A3A1C09B613E8DEE3D05006.tpdila13v_2?idArticle=LEGIARTI000029122652&cidTexte=LEGITEXT000005633251&dateTexte=20150331) ([ainsi que décret afférent](http://www.legifrance.gouv.fr/affichTexte.do;jsessionid=6CF268E473632E1932F55A719251387B.tpdjo12v_2&dateTexte=?cidTexte=JORFTEXT000000413818&categorieLien=cid)).

Je cite:

> Des licences de réutilisation des données mentionnées à l'article 1er et détenues par l'Etat peuvent être accordées aux personnes qui souhaitent faire usage de ces données dans le cadre de leur activité, que celle-ci ait ou non un caractère commercial. [...]

Cela s'applique en particulier aux données issues de Legifrance / data.gouv, mais il ne me semble a priori pas absurde d'envisager que la diffusion qui en est faite ici soit couverte.

**Comment by steeve** [comment]
@hugoroy
Je supposais que c'etait le cas, mais dans le doute j'ai rajoute le CC
tu penses que je peux l'enlever, ou il existe un texte legal plus adapte ?

**Comment by lisael** [comment]
sous reserve de ton `[...]`, je cite:

> Une convention précise les conditions d'utilisation des données et, notamment, les engagements pris par le bénéficiaire afin de garantir que l'usage qui en sera fait répond à l'exigence de fiabilité qui s'impose pour la diffusion de telles données.

**Comment by hugoroy** [comment]
Je ne sais pas comment les données ont été récupérées, d'où elles viennent etc. Mais si ça vient de légifrance, la "licence ouverte" s'applique (edit: il me semble, j'ai regardé rapidement). Il suffit de rajouter dans un texte quelque part (par exemple le README)

```
En application de la "licence ouverte" Etalab 1.0, les informations reproduites ont pour origine [SOURCE][LIEN], à la date de mise à jour du DATE.
```

Rajouter une licence Creative Commons me semble problématique (pas forcément d'un point de vue légal, mais plutôt éthique).

**Comment by steeve** [comment]
J'imagine que se pose aussi la question des erreurs dans le scraping des articles.

A la base, chaque article avait son lien legifrance, mais ca rendait les diff indigestes, ce qui casse un peu l'idee, du coup.

@hugoroy: quoi ca te semble problematique d'un point de vue ethique? si tu as une autre licence (si licence il y a besoin), je suis preneur

**Comment by hugoroy** [comment]
problématique parce que la licence fait croire qu'il y a des droits à donner en licence (des droits d'auteur) là où, en réalité, il n'y en a pas. A la limite, le seul contenu original qui est donné en licence est le README... autant le mettre dans une licence comme la CC-0, plus proche du domaine public.

de mon point de vue, ce genre de données et de texte doit être complètement dans le domaine public.

Aucun problème pour les modifications des articles, la licence l'autorise expressément.

**Comment by rakoo** [comment]
Je suis d'accord avec hugoroy pour utiliser la CC0 à defaut du domaine public, d'autant plus que techniquement une "Licence Creative Commons" tout court n'existe pas, en l’état c'est la CC-BY qui est utilisée. En plus de ça la CC-BY oblige un tiers a citer le nom de l'auteur.

**Comment by eric-brechemier** [comment]
**Note :** la "License Ouverte" d'Etalab a été conçue pour être compatible avec les licenses Creative Commons :

> Pour faciliter la réutilisation des « Informations », cette licence a été conçue
> pour être compatible avec toute licence libre qui exige
> a minima la mention de paternité.
> Elle est notamment compatible avec les licences « Open Government Licence » (OGL)
> du Royaume-Uni, « Creative Commons Attribution 2.0 » (CC-BY 2.0) de Creative Commons
> et « Open Data Commons Attribution » (ODC-BY) de l’Open Knowledge Foundation. 

http://www.etalab.gouv.fr/licence-ouverte-open-licence

**Comment by mems** [comment]
C'est en effet un point à vérifier. Un lecteur de Numérama aurait eu des difficultés à ce sujet sur un projet similaire : http://www.numerama.com/magazine/32646-le-code-civil-sous-github-pour-mieux-lire-ses-evolutions.html#ac_com1749221

**Comment by nclsgd** [comment]
Je m'étais également intéressé à la transcription des textes de loi de Légifrance sur Git il y a un an ou deux et il y avait à l'époque un problème légal avec le _scraping_ de Légifrance (je pense d'ailleurs que c'est toujours limite) et le téléchargement de la base de données **LEGI** (les textes de lois) était réservé aux possesseurs d'une licence payante[1] délivrée par la DILA.

IANAL mais ce souci semble avoir été corrigé avec par _l'arrêté du 24 juin 2014 relatif à la gratuité de la réutilisation des bases de données juridiques et associatives de la direction de l'information légale et administrative_ (http://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000029135221&fastPos=1&fastReqId=1203444049&categorieLien=cid&oldAction=rechTexte).

Et effectivement, la DILA a mis à disposition leurs bases de données (**JORF**, **JORFSIMPLE**, **LEGI**, **KALI**, **CNIL**, **CONSTIT** et **CIRCULAIRES**) ainsi que leurs mises à jour sur un serveur FTP public : http://rip.journal-officiel.gouv.fr/index.php/pages/juridiques

Concernant la licence du contenu, celle-ci doit répondre à ce texte : http://rip.journal-officiel.gouv.fr/index.php/pages/LO

[1]: Pour rire, les tarifs sont toujours disponibles : http://rip.journal-officiel.gouv.fr/index.php/content/download/531/2797/file/Tarifs%20du%20RIP%20V2_2013_01_01.pdf

**Comment by eric-brechemier** [comment]
L'article pertinent, en suivant de proche en proche la référence donnée par @mems:

> Des licences de réutilisation des données mentionnées à l'article 1er et détenues par l'Etat peuvent être accordées aux personnes qui souhaitent faire usage de ces données dans le cadre de leur activité, que celle-ci ait ou non un caractère commercial. Une convention précise les conditions d'utilisation des données et, notamment, les engagements pris par le bénéficiaire afin de garantir que l'usage qui en sera fait répond à l'exigence de fiabilité qui s'impose pour la diffusion de telles données.
> 
> La décision d'accorder la licence est prise par l'autorité responsable de l'exploitation du site sur lequel sont diffusées les données objet de la licence.
> 
> Les licences sont accordées à titre gracieux. Les licences ne peuvent être rétrocédées.
1. http://rip.journal-officiel.gouv.fr/index.php/pages/juridiques
2. http://rip.journal-officiel.gouv.fr/index.php/pages/LO
3. http://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000413818&dateTexte=20140703
4. http://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000413818&dateTexte=20140703#LEGIARTI000029122652

**Comment by eric-brechemier** [comment]
> [1]: Pour rire, les tarifs sont toujours disponibles : http://rip.journal-officiel.gouv.fr/index.php/content/download/531/2797/file/Tarifs%20du%20RIP%20V2_2013_01_01.pdf

@nicolasgodinho "La référence au service du public"

**Comment by nclsgd** [comment]
J'ai oublié qu'il faudrait aussi mentionner l'origine et la date de récupération des informations (idéalement dans les messages de commit Git) pour répondre à cette exigence de la DILA : http://rip.journal-officiel.gouv.fr/index.php/pages/mention#mension

_P.S._ ~~Je n'ai pas compris leur histoire d'URL courte... Sachant que le format d'URL de Légifrance est juste attroce et que je n'ai jamais réussi à récupérer une URL vraiment _courte_ pointant correctement vers un article ou un texte de loi sans me faire rediriger vers une autre page de Légifrance, je ne vois pas vraiment comment produire une URL courte dans un message de commit...~~
Bon, en fait si ça a l'air de fonctionner correctement avec ce format : http://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000029135221

**Comment by ghost** [comment]
Pour info, mes mésaventures avec la DILA date d'environ fin 2011. A cette époque, Etalab était tout jeune (et méconnu). Surtout, comme le souligne @nicolasgodinho, il n'y avait pas de possibilité légale de réutiliser les bases de la DILA, et le scraping était totalement illégal à leurs yeux.

A cette époque, une discussion sur linuxfr.org (que je n'arrive pas à retrouver) avait conclu que l'importation du Code civil sous forme de base git était interdite.

Visiblement, de l'eau a coulé sous les ponts, et il est fort possible que les mésaventures auxquelles j'ai fait face ne soient absolument plus valables aujourd'hui.

Néanmoins, je conseille aux mainteneurs du projet de prendre au sérieux ce bug. S'il y a bien un bug qui pourrait faire foirer tout le projet s'il est mal résolu, c'est bien celui-ci.

Vag'

**Comment by mickaelandrieu** [comment]
De toute façon tout ce qui relève du "Domaine public" ne peut être licencé, cette discussion est un non sens.

Le tout étant de savoir si les lois relève vraiment du domaine public: ne pas avoir d'auteur ne veut pas dire relever du domaine public, puisque le domaine public en France s'applique après période fixe après la mort de la créateur d'une oeuvre et qu'on ne _peut_ pas téléverser une oeuvre dans le domaine public.

Ce qui peut être mis sous licence open source, c'est le travail de compilation des documents. La question que je me pose, c'est est-ce qu'on a le droit d'altérer le contenu des lois ?

Ce serait une bonne question à poser à l'assemblée nationale :-)

**Comment by ghost** [comment]
@mickaelandrieu Cette discussion avait un sens avant le 24 juin 2014, puisqu'un projet comme celui-ci était tout simplement illégal sans paiement d'une redevance auprès de la DILA. Je pense que cela reste très utile de confirmer la légalité d'un travail de compilation sur github auprès de la DILA.

Ceci fait, le résultat de cette compilation et du suivi sous forme de commits ne peut pas être mise sous une quelconque licence, en effet.

**Comment by nclsgd** [comment]
> Néanmoins, je conseille aux mainteneurs du projet de prendre au sérieux ce bug. S'il y a bien un bug qui pourrait faire foirer tout le projet s'il est mal résolu, c'est bien celui-ci.

Entièrement d'accord avec @Vagdish.
Et sans vouloir jouer les rabat-joie mais je doute un peu de la légalité de la mise en ligne voire même de l'utilisation d'un _scraper_ pour Légifrance...

> De toute façon tout ce qui relève du "Domaine public" ne peut être licencé, cette discussion est un non sens.

@mickaelandrieu: Pas vraiment. IANAL mais il me semble que ce qui est licencié ici n'est pas les textes de lois en eux-même mais plutôt le travail de compilation et de retranscription sous format XML par la DILA (et dont Légifrance n'est qu'une interface d'exploration).
C'est le problème de l'œuvre dérivée en droit de la propriété intellectuelle. Enfin du moins, c'est comme ça que je l'ai compris. J'espère juste ne pas trop me planter en écrivant ça... Un juriste pour confirmer ou infirmer ? (cc @hugoroy)

**Comment by mickaelandrieu** [comment]
@nicolasgodinho @Vagdish je plussois, l'avis d'un juriste m'intéresse beaucoup ça dépasse mes (maigres) connaissances sur la question (je suis évangéliste open source).

**Comment by hugoroy** [comment]
Non, il ne s'agit pas d'un problème de droit d'auteur. Les textes de lois ne sont pas soumis au droit d'auteur, ils sont du domaine public. C'est une jurisprudence constante (voir notamment l'arrêt de la chambre criminelle de la cour de cassation du 2 juin 1982 – pourvoi n° 81-92.330), en tout cas je n'ai pas connaissance d'arrêt qui ait remis en cause ce principe. (Fort heureusement d'ailleurs, car des professions juridiques entières ne pourraient plus travailler convenablement...)

S'il y a des restrictions légales, c'est du droit sui generis des bases de données qu'il viendrait. Je ne sais pas comment les données ici ont été récupérées, mais comme indiqué, si les données viennent de légifrance, alors la licence ouverte s'applique (voir les "mentions" qui renvoient à la DILA qui renvoie à la "licence ouverte" Etalab 1.0). Il suffit de mentionner ce que j'ai indiqué  là https://github.com/steeve/france.code-civil/issues/5#issuecomment-88038255

Hormis les informations venant de légifrance, je ne vois que le "README". Ce texte peut être mis sous une licence CC-BY (même si je pense qu'une CC-0 serait plus adaptée, mais le choix ne m'appartient pas...)

S'il y a des éléments protégeables par le droit d'auteur, ce serait plutôt les programmes écrits dans le but de récupérer, organiser, éditer et publier les "informations" récupérées. (une licence de logiciel libre suffirait, par exemple:  MIT ou GPL)

**Comment by RouxRC** [comment]
En l'état réutiliser les données de la loi est devenu légal depuis le milieu de l'année passée, mais il reste une faille dans le [décret de 2002](http://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000413818) créant une certaine insécurité juridique
Plus de détails ici : http://www.regardscitoyens.org/apprenons-des-echecs-de-la-dila-episode-1-comment-faire-de-lopen-data/

**Comment by hugoroy** [comment]
Sauf erreur de ma part, cette "issue" n'a toujours pas été clarifiée. Besoin d'aide @steeve ?

**Comment by steeve** [comment]
indeed, mais est-ce toujours un probleme ?

**Comment by RouxRC** [comment]
le décret n'a malheureusement toujours pas été corrigé malgré des demandes répétées...
